src/pudl_archiver/archivers/epacems.py

```diff
@@ -66,7 +64,9 @@
     "wi",
     "wy",
 ]
-BASE_URL = "https://gaftp.epa.gov/DMDnLoad/emissions/hourly/monthly"
+
+BASE_URL = "https://api.epa.gov/easey/bulk-files/emissions/hourly/state/"
+parameters = {"api_key": os.environ["EPACEMS_API_KEY"]}  # Set to API key
```

Comment:
How do the parameters / API key get passed through to the API request?

Reply:
Also this would be a great kind of thing to check for in the archive validations, and it's probably not too hard.

Or maybe this should be a more generic check that's done in the requests for all file downloads if we aren't already -- if we get a 404 or a 403 or some other error code, then probably something is wrong.

Reply:
Oh I guess it's an API error not an HTTP error response so it's dataset specific.

```xml
<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>AccessDenied</Code>
  <Message>Access Denied</Message>
  <RequestId>FWYCXG03J3B1X2F4</RequestId>
  <HostId>xP3qPNcx25yKpg1tNpVPQ+39RPCTG1VSpnmU1UraUfm91CXfACUaJlUgl6nm04zMh+wdgBOBJONGmx5Tcx0Ayw==</HostId>
</Error>
```

Reply:
Great question. This works as is because turns out you can query the file downloads directly without needing an API - the API is only used if you're asking for details about files, which we aren't because of the way I've constructed this here. So perhaps I will deprecate the use of the API altogether in this archiver and see if it continues to run as expected.

Reply:
If you just try and download one of these CSVs directly in a browser or using `wget` or `curl` it works? That seems surprising to me.  In their example they're using the `requests` library, and are setting the parameters dictionary such that it includes the API key, which I think means it should be appending `?api_key=siodfsjdhkajsdskskldjdlkjf` to the URL that's actually accessed when they use `request.get()`

Reply:
Well, downloading with `wget` worked fine locally!

```bash
wget "https://api.epa.gov/easey/bulk-files/emissions/hourly/state/emissions-hourly-2020-vt.csv"
```

But I don't know what about the API there is to remove from this archiver. It's currently just downloading from a URL directly, and not attempting to use the API key.

Reply:
Exactly what I mean, I'm just going to clean it up from the various places I've added it to (tox, etc.). Even more straightforward than it seems.